mage/packages.go

```diff
@@ -0,0 +1,982 @@
+		version:    appIngressVersion,
+		gitBranch:  appIngressBranch,
+		gitRepo:    appIngressRepo,
+	}
+
+	pkgStorageApp = gravityPackage{
+		repository: "gravitational.io",
+		name:       "storage-app",
+		version:    appStorageVersion,
+		gitBranch:  appStorageBranch,
+		gitRepo:    appStorageRepo,
+	}
+
+	pkgTillerApp = gravityPackage{
+		repository: "gravitational.io",
+		name:       "tiller-app",
+		version:    appTillerVersion,
+		setImage:   []string{fmt.Sprint("gcr.io/kubernetes-helm/tiller:v", tillerVersion)},
+		include:    []string{"resources", "registry"},
+		srcDir:     "assets/tiller-app",
+	}
+
+	pkgRBAC = gravityPackage{
+		repository: "gravitational.io",
+		name:       "rbac-app",
+		version:    appRBACVersion,
+		include:    []string{"resources", "registry"},
+		srcDir:     "assets/rbac-app",
+	}
+
+	pkgDNSApp = gravityPackage{
+		repository: "gravitational.io",
+		name:       "dns-app",
+		version:    appDNSVersion,
+		setImage:   []string{fmt.Sprint("dns-app-hooks:", appDNSVersion)},
+		include:    []string{"resources", "registry"},
+		srcDir:     "assets/dns-app",
+	}
+
+	pkgBandwagonApp = gravityPackage{
+		repository: "gravitational.io",
+		name:       "bandwagon",
+		version:    appBandwagonVersion,
+		gitBranch:  appBandwagonBranch,
+		gitRepo:    appBandwagonRepo,
+	}
+
+	pkgTelekube = gravityPackage{
+		repository: "gravitational.io",
+		name:       "telekube",
+		version:    buildVersion,
+		srcDir:     "assets/telekube",
+		env: map[string]string{
+			"GRAVITY_K8S_VERSION": k8sVersion,
+		},
+		force: true,
+	}
+
+	pkgKubernetes = gravityPackage{
+		repository: "gravitational.io",
+		name:       "kubernetes",
+		version:    buildVersion,
+		exclude:    []string{"**/*.tf"},
+		srcDir:     "assets/kubernetes",
+		deps: []string{
+			pkgGravity.Locator(),
+			pkgTeleport.Locator(),
+			pkgFio.Locator(),
+			pkgPlanet.Locator(),
+			pkgWebAssets.Locator(),
+			pkgRBAC.Locator(),
+			pkgDNSApp.Locator(),
+			pkgLoggingApp.Locator(),
+			pkgMonitoringApp.Locator(),
+			pkgIngressApp.Locator(),
+			pkgStorageApp.Locator(),
+			pkgBandwagonApp.Locator(),
+			pkgTillerApp.Locator(),
+			pkgSiteApp.Locator(),
+		},
+		force: true,
+	}
+)
+
+func (Package) Telekube() (err error) {
+	mg.Deps(Build.Go, Package.K8s)
+
+	m := root.Target("package:telekube")
+	defer func() { m.Complete(err) }()
+
+	return trace.Wrap(pkgTelekube.localAppImport(m, consistentStateDir()))
+}
+
+func (Package) K8s() (err error) {
+	mg.Deps(Build.Go, Package.GravityPackage, Package.Teleport, Package.Fio, Package.Planet, Package.Web,
+		Package.Site, Package.Monitoring, Package.Logging, Package.Ingress, Package.Storage, Package.Tiller,
+		Package.Rbac, Package.DNS, Package.Bandwagon)
+
+	m := root.Target("package:k8s")
+	defer func() { m.Complete(err) }()
+
+	return trace.Wrap(pkgKubernetes.localAppImport(m, consistentStateDir()))
+}
+
+func (Package) GravityPackage() (err error) {
+	mg.Deps(Build.Go)
+
+	m := root.Target("package:gravity-package")
```

Comment:
Hmm, so we have packages (gravity, planet) and apps (site, monitoring, etc) so I wonder if using the same naming in targets would make sense. I.e. this target name would be `package:gravity`. Targets above would be `app:telekube`, `app:k8s`, `app:bandwagon`, etc.